sdk/storage/Azure.Storage.Blobs/src/BlobExtensions.cs

```diff
@@ -92,5 +92,54 @@ internal static string ToTagsString(this Tags tags)
             }
             return string.Join("&", encodedTags);
         }
+
+        /// <summary>
+        /// Creates a new BlobProperties object backed by BlobPropertiesInternal.
+        /// </summary>
+        /// <param name="properties">The BlobPropertiesInternal returned with the request</param>
+        internal static BlobProperties ToBlobProperties(this BlobPropertiesInternal properties) =>
```

Comment:
```this BlobPropertiesInternal properties``` using the ```this``` keyword, we can make this static method an [extension method](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/extension-methods).